gui/src/ts/api/queries/benchmark/BenchmarkAnswerPageQuery.graphql

```diff
@@ -23,8 +23,16 @@ query BenchmarkAnswerPageQuery(
             choiceAnalyses {
               choiceLabel
               questionAnswerPaths {
-                endNodeId
                 startNodeId
+                startNode {
+                  id
+                  label
+                }
+                endNodeId
+                endNode {
+                  id
+                  label
+                }
                 paths {
                   path
```

Comment:
You have few enough paths on this single page that you can use the `edges` field. It has redundancy but it allows you to get the node labels.

I like that the graph shows id's, though, since the id's have things like WordNet synsets encoded in them.
The table to the right of each graph should be something like a KgNodeTable but with nodes and edges. Node label, pos, ...

Can be done in another PR.